charts/ccd-data-store-api/values.yaml

```diff
@@ -89,3 +89,7 @@ java:
     DATA_STORE_CROSS_JURISDICTIONAL_ROLES: caseworker-caa,caseworker-approver
     DATA_STORE_CITIZEN_ROLES: citizen,letter-holder
     DATA_STORE_SECURITY_LOGGING_FILTER_PATH_REGEX: '.*'
+
+    AM_ROLE_ASSIGNMENT_PAGINATION_ENABLED: false
+    AM_ROLE_ASSIGNMENT_STARTING_PAGE_NUMBER: 0
```

Comment:
That's correct, Mutlu. I was thinking, in case RAS team decides to change the page starting number to 1 with a help of spring attribute (spring.data.web.pageable.one-indexed-parameters=true), this config will come in handy.

Reply:
Moved to a constant.